Documentation/configuration/metrics.rst

```diff
@@ -42,6 +42,12 @@ Datapath
 --------
 
 * ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_expired_total``: Number of deleted entries in the conntrack table by the garbage collector process
```

Comment:
Are there any cases where we might delete an entry that isn't expired? An example would be if we are cleaning up entries for a non-existent endpoint? If so, perhaps calling this `datapath_conntrack_gc_deleted_total` and then using a label for the reason would make it easier to split out the various scenarios.

Reply:
Good point. I'm fine with this as well